extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzScheduler.java

```diff
@@ -68,7 +68,8 @@
     }
 
     public QuartzScheduler(SchedulerContext context, QuartzSupport quartzSupport, Config config) {
-        if (!quartzSupport.getRuntimeConfig().forceStart && context.getScheduledMethods().isEmpty()) {
+        if ((!quartzSupport.getRuntimeConfig().forceStart && context.getScheduledMethods().isEmpty())
+                || !config.getValue("quarkus.quartz.enabled", Boolean.class)) {
```

Comment:
As I commented on the issue - the preferred solution is to inject the config group directly.

Reply:
Ok, I tried to inject the runtime config as a variable into QuartzScheduler. Here the tests failed because it was always null. But then I injected it into the constructor and now the tests pass. 
But if I use the extension directly in any project now, I get those errors with `Unsatisfied dependency for type io.quarkus.scheduler.runtime.SchedulerRuntimeConfig and qualifiers [@Default]`

Reply:
> But then I injected it into the constructor and now the tests pass.

That's weird. Constructor and field injection should behave identically. I'll try to change the code locally...

> But if I use the extension directly in any project now...

Does it happen in both the production and the dev mode?

Reply:
So I've tried to change the code to use field injection (i.e. `@Inject SchedulerRuntimeConfig config`) and it works fine. However, constructor injection is more appropriate in this particular use case (we don't need to store the config in a field ;-).

Reply:
Thanks for investigating! I guess it is then some local mistake on my side.
Then I will leave it as injected constructor param.

Reply:
I've also verified that it works with both the dev and the prod mode using the [scheduler-quickstart](https://github.com/quarkusio/quarkus-quickstarts/tree/master/scheduler-quickstart)